### PR TITLE
[#166] Add VanillaJS "01-login" quickstart guide

### PR DESCRIPTION
**Pull request by harenson**
Add VanillaJS `01-login` quickstart guide.

### CODE REVIEW

**Comment by auth0build** [comment]
Can one of the admins verify this patch?

File: articles/client-platforms/vanillajs/01-login.md [hunk 1]
```diff
@@ -1,125 +1,163 @@
+
 ---
 title: Login
-description: This tutorial will show you how to use the Auth0 Javascript SDK to add authentication and authorization to your web app.
+description: This tutorial will show you how to use the Auth0 to add authentication and authorization to your VanillaJS app.
```
Comment (sildur) [inline articles/client-platforms/vanillajs/01-login.md]: "how to use Auth0"

**Comment by sildur** [comment]
:+1: @chenkie this can be merged.

**Comment by harenson** [comment]
Included in https://github.com/auth0/docs/pull/2168

**Pull request closed without merging**